packages/react/src/components/ContentGroupCards/__stories__/ContentGroupCards.stories.js

```diff
@@ -15,18 +15,26 @@ export default {
 
   parameters: {
     ...readme.parameters,
+    knobs: {
+      ContentGroupCards: ({ groupId }) => ({
+        heading: text('Heading (heading):', ContentGroupCardsKnobs.heading, groupId),
+        copy: text('Copy (copy):', ContentGroupCardsKnobs.copy, groupId),
+        items: object('Content group cards (Items) :', ContentGroupCardsKnobs.items, groupId),
```

Comment:
hey @greeshmareddyg , thank you for this! However, we are moving away from having large JSON objects passed in as knobs. Perhaps just remove the `object` knob for now and just pass in the `ContentGroupCardsKnobs.items` directly, and we will figure out a cleaner way to author the items without using JSON in another PR.

Reply:
@jeffchew I've removed the object and passed the items directly.